src/main/resources/reference.conf

```diff
@@ -862,17 +878,27 @@ resourceTypes = {
     }
     ownerRoleName = "creator"
     roles = {
+      # the creator role is assigned directly to the persistent-disk resource and does not include can_access
+      # but can_access is required to perform any action so must be granted via the parent resource using
+      # either the manager or accessor role
       creator = {
         roleActions = ["read", "attach", "modify", "delete", "read_policies", "set_parent", "get_parent"]
       }
       manager = {
-        roleActions = ["delete", "read", "read_policies"]
+        roleActions = ["delete", "read", "read_policies", "can_access"]
+      }
+      accessor = {
+        roleActions = ["can_access"]
       }
     }
     reuseIds = false
+    prerequisiteAction = "can_access"
   }
   kubernetes-app = {
```

Comment:
I assume you mean `kubernetes-app-shared` and if so, no because _all_ permissions are already inherited. The other resources behave differently because there is a `creator` role that is assigned directly to a single user.

Reply:
actually, you were right, I missed it